specifications/api/swagger.yaml

```diff
@@ -35,12 +35,27 @@ numberOfSlots: &numberOfSlots
   properties:
     quantity:
       type: integer
-      minimum: 0
+      minimum: 1337
     unit:
       type: string
       enum:
         - slot
-      example: 1337
+      example: "slot"
+
+numberOfBlocks: &numberOfBlocks
+  type: object
+  required:
+    - quantity
+    - unit
+  properties:
+    quantity:
+      type: integer
+      minimum: 1337
```

Comment:
minimum ought to be 0 here too, but we can add an `example: 1337`

Reply:
:duck: 

yes - will fix